CHANGELOG.md

```diff
@@ -6,11 +6,15 @@
   `SimulatorDevice` class for general IonQ devices.
   [(#50)](https://github.com/PennyLaneAI/PennyLane-IonQ/pull/50)
 
+### Bug Fixes
+
+* Since the probabilities returned don't always perfectly sum to one, they are now normalized.
```

Comment:
```suggestion
* Since the probabilities returned don't always perfectly sum to one, they are now normalized.
  [(#53)](https://github.com/PennyLaneAI/PennyLane-IonQ/pull/53)
```